raiden-dapp/src/components/ConnectionManager.vue

```diff
@@ -137,6 +145,18 @@ export default class ConnectionManager extends Vue {
     return DirectRpcProvider.isAvailable;
   }
 
+  async created(): Promise<void> {
+    this.checkProvidersForBeingDisabled();
+  }
+
+  async checkProvidersForBeingDisabled(): Promise<void> {
```

Comment:
`checkDisabledProviders`

Reply:
I would like to insist on my naming. Yours is more like "checking the disabled providers". But it is checking all providers to see which of them are disabled.

Reply:
Sure, as I already approved, feel free to go with this naming  but fyi it is grammatically wrong :) `checkForDisbaledProviders` would be more correct too say the same thing.

Reply:
Perfect. Thanks! 🙏🏾

Reply:
✔️